.github/workflows/CI.yml

```diff
@@ -31,6 +31,11 @@ jobs:
           apt install --yes --no-install-recommends ca-certificates
 
           apt install --yes --no-install-recommends dotnet8
+          
+          # trust ASP.NET Core HTTPS development certificate
```

Comment:
@webwarrior-ws please add the *why*, not just the *what*

Reply:
updated comment